pna.p4

```diff
@@ -665,19 +665,75 @@ extern void mirror_packet(MirrorSlotId_t mirror_slot_id,
 // probably be a parameter that specifies the new entry's initial
 // expire_time_profile_id.
 
-extern bool add_entry<T>(string action_name,
-                         in T action_params,
-                         in ExpireTimeProfileId_t expire_time_profile_id);
-
-// If we support the following variant of add_entry, with no
-// expire_time_profile_id parameter, then we need to define its
-// behavior in terms of what the added entry's value of
-// expire_time_profile_id is.  One option would be to define the
-// behavior to be the same as the 3-parameter version of add_entry
-// above, with expire_time_profile_id equal to 0.
-
-//extern bool add_entry<T>(string action_name,
-//                         in T action_params);
+// The bit width of this type is allowed to be different for different
+// target devices.  It must be at least a 1-bit wide type.
+
+typedef bit<1> AddEntryErrorStatus_t;
+
+const AddEntryErrorStatus_t ADD_ENTRY_SUCCESS = 0;
+const AddEntryErrorStatus_t ADD_ENTRY_NOT_DONE = 1;
+
+// Targets may define target-specific non-0 constants of type
+// AddEntryErrorStatus_t if they wish.
+
+// add_entry() must be called within the body of a P4 action that is
+// the default_action of a table with the table property `add_on_miss`
```

Comment:
should we not show in the example, the usage of the property 'add_on_miss = true' ?

Reply:
There is such an example, here: https://github.com/p4lang/pna/blob/main/examples/pna-example-tcp-connection-tracking.p4#L273

Is there some other example program you are referring to htat is missing `add_on_miss = true`?